requirements_dev.txt

```diff
@@ -0,0 +1,14 @@
+pip
+bump2version
+wheel
+watchdog
+flake8
+tox
+coverage
+Sphinx
+twine
+Click
+black
+omop2obo==1.0.0
```

Comment:
any reason to set this? I suspect I will be pushing new releases soon that will extend the functionality of v1.0.0